README.md

```diff
@@ -47,6 +47,9 @@ In this workflow switch owner verifies device's Initial Attestation Key (IAK) an
 fully control certificate structure, revocation and expiration policies and (2) remove external dependency on switch vendor CA during TPM attestation workflow. The assumption is that before the device is shipped to the switch owner, a switch vendor provisions each control card with IAK and IDevID certificates following the TCG specification in
 [Section 5.2](https://trustedcomputinggroup.org/wp-content/uploads/TPM-2p0-Keys-for-Device-Identity-and-Attestation_v1_r12_pub10082021.pdf#page=20) and [Section 6.2](https://trustedcomputinggroup.org/wp-content/uploads/TPM-2p0-Keys-for-Device-Identity-and-Attestation_v1_r12_pub10082021.pdf#page=30).
 
+Switch vendors must use the strongest asymmetric cryptographic algorithms supported by the TPM 2.0 to create IAK and IDevID key pairs, which implies relying on ECC P521 or, less preferably, RSA 4096.
+The same expectations apply to the switch vendor CA keys used to sign IAK and IDevID certificates.
+
```

Comment:
Why not just say 

Vendors must use one of the follow modes for generating IAK and IDevID keypairs as well as CA signing keys.
* ECC P521 (preferred)
* RSA 4096

Reply:
Sounds good, please take a look.